src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -558,6 +558,175 @@ def test_ppmu_source(multi_instrument_session):
+    multi_instrument_session.create_time_set(time_set_b)
+    assert multi_instrument_session.get_time_set_name(0) == time_set_a
+    assert multi_instrument_session.get_time_set_name(1) == time_set_b
+    multi_instrument_session.delete_all_time_sets()
+    time_set = None
+    try:
+        time_set = multi_instrument_session.get_time_set_name(0)
+    except Exception:
+        pass
+    assert time_set is None
+
+
+def test_configure_get_time_set_period(multi_instrument_session):
+    # Tests configure_time_set_period()
+    # Tests get_time_set_period()
+    time_set_name = 'time_set_abc'
+    time_set_period = datetime.timedelta(microseconds=10)
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.create_time_set(time_set_name)
+    assert multi_instrument_session.get_time_set_period(time_set_name) == 1e-6
+    multi_instrument_session.configure_time_set_period(time_set_name, time_set_period)
+    assert multi_instrument_session.get_time_set_period(time_set_name) == time_set_period.total_seconds()
+
+
+def test_configure_get_time_set_drive_format(multi_instrument_session):
+    # Tests configure_time_set_drive_format()
+    # Tests get_time_set_drive_format()
+    time_set_name = 'time_set_abc'
+    time_set_drive_format = nidigital.DriveFormat.SBC
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.create_time_set(time_set_name)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_drive_format(time_set_name) == nidigital.DriveFormat.NR
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].configure_time_set_drive_format(time_set_name, time_set_drive_format)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_drive_format(time_set_name) == time_set_drive_format
+
+
+def test_configure_get_time_set_edge(multi_instrument_session):
+    # Tests configure_time_set_edge()
+    # Tests get_time_set_edge()
+    time_set_name = 'time_set_abc'
+    time_set_period = datetime.timedelta(microseconds=10)
+    time_set_drive_on = time_set_period * 0.5
+    multi_instrument_session.load_pin_map(os.path.join(test_files_base_dir, "pin_map.pinmap"))
+
+    multi_instrument_session.create_time_set(time_set_name)
+    multi_instrument_session.configure_time_set_period(time_set_name, time_set_period)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_edge(time_set_name, nidigital.TimeSetEdgeType.DRIVE_ON) == 0
+    multi_instrument_session.pins['site0/PinA', 'site1/PinC'].configure_time_set_edge(time_set_name, nidigital.TimeSetEdgeType.DRIVE_ON, time_set_drive_on)
+    assert multi_instrument_session.pins['site0/PinA', 'site1/PinC'].get_time_set_edge(time_set_name, nidigital.TimeSetEdgeType.DRIVE_ON) == time_set_drive_on.total_seconds()
+
+
+def test_time_set_edges(multi_instrument_session):
```

Comment:
`test_configure_time_set_edges`?

Reply:
This is no longer an issue, because the test has been split and the new tests have different names.